app/utils/AddSealHelper.scala

```diff
@@ -54,6 +54,7 @@ class AddSealHelper(userAnswers: UserAnswers) {
       val numberOfSeals = userAnswers.get(DeriveNumberOfSeals()).getOrElse(0)
 
       val singularOrPlural = if (numberOfSeals == 1) "singular" else "plural"
+      val idPluralisation  = if (numberOfSeals == 1) "" else "s"
```

Comment:
To avoid any interpolation and make it easy to look for ids can we do:
```suggestion
      val idPluralisation  = if (numberOfSeals == 1) "change-seal" else "change-seals"
```
?

Reply:
👍